source/containers.tex

```diff
@@ -14895,8 +14895,8 @@
       { x.swap(y); }
 
   private:
-    containers c;               // \expos
-    key_compare compare;        // \expos
+    containers \exposid{c};               // \expos
+    key_compare \exposid{compare};        // \expos
 
     struct key_equiv {  // \expos
```

Comment:
I think `key_equiv` should also be formatted with `\exposid{}`. Perhaps its `comp` member should also be considered exposition only, although not strictly needed due to `priority_queue`'s `comp`.

(How about changing to kebab-case and/or eliminating its ctor as aggregate initialization can do the right thing?)